templates/000-feature-template.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# <RFC title>
+
+## Summary
+
+> One paragraph explanation of the feature.
+
+## Motivation
+
+> Why are we doing this? What use cases does it support? What is the expected outcome?
+
+## Guide-level explanation
+
+> Explain the proposal as if it was already included in Amundsen and you were teaching it to an Amundsen user. That generally means:
+
+> Introducing new named concepts.
+> Explaining the feature largely in terms of examples.
+> Explaining how Amundsen users should think about the feature, and how it should impact the way they use Amundsen. It should explain the impact as concretely as possible.
+> If applicable, provide deprecation warnings, or migration guidance.
+> For implementation-oriented RFCs, this section should focus on how maintainers should think about the change, and give examples of its concrete impact. For policy RFCs, this section should provide an example-driven introduction to the policy, and explain its impact in concrete terms.
+
+## Reference-level explanation
```

Comment:
It'd be good to have Guide-level, design-level and reference-level as three headings.
Also, I'd put the alternative names besides these:
Guide-level explanation (aka Product details)
Reference-level explanation (aka Technical details)

Reply:
Sounds good, will add!